markdown-mode.el

```diff
@@ -6527,6 +6527,46 @@ BEG and END are the limits of scanned region."
        (progn (goto-char end) (forward-line 1) (point))))))
 
 
+;;; Display inline image =================================================
+
+(defvar markdown-inline-image-overlays nil)
+(make-variable-buffer-local 'markdown-inline-image-overlays)
+
+(defun markdown-remove-inline-images ()
+  (interactive)
+  (mapc 'delete-overlay markdown-inline-image-overlays)
+  (setq markdown-inline-image-overlays nil))
+
+(defun markdown-display-inline-images ()
+  (interactive)
+  (unless (display-graphic-p)
+    (error "Cannot show images."))
+  (save-excursion
+    (save-restriction
+      (widen)
```

Comment:
Do you need to widen here? 

My question's context: I'm thinking for my use-case I'll probably narrow to region of the day I'm writing in, and only want the pictures toggles for that day (toggling them for all days could make things slow, particularly because I intend to hook into this code & use ImageMagik to make the photos no wider than the buffer size).